### PR TITLE
Update Navigation menu tags to render correct breadcrumbs . 

### PR DESCRIPTION
**Pull request by NarenderRajuB**
Feature/opencms 932:
1. Add is_home attribute to the Menu item node.
2. Update show breadcrumb navigation for navigation to ignore empty content navigation nodes

### CODE REVIEW

**Review by Aiky30 — CHANGES_REQUESTED** [review]

File: tests/test_cms_menus.py [hunk 1]
```diff
@@ -810,6 +810,99 @@ def test_basic_projects_softroot_rendering_nodes(self):
...
+    def test_show_breadcrumb(self):
+        """
+        Tree in fixture :
+            menuroot
+               aaa ( Home page Node)
+                   aaa1
+                       ccc
+                           ddd
+                   aaa2
+               bbb
```
[the remark this answers is not in the document]
Comment (NarenderRajuB) [reply tests/test_cms_menus.py:822]: Updated . please review

File: tests/test_cms_menus.py [hunk 2]
```diff
@@ -810,6 +810,99 @@ def test_basic_projects_softroot_rendering_nodes(self):
...
+        self.assertEqual(len(nodes), 3)
+
+        tpl = Template("{% load navigation_menu_tags %}{% show_breadcrumb 1 %}")
+        tpl.render(context)
+        nodes = context['ancestors']
+
+        self.assertEqual(len(nodes), 2)
+
+        context = self.get_context()
+        tpl = Template("{% load navigation_menu_tags %}{% show_breadcrumb %}")
```
[the remark this answers is not in the document]
Comment (NarenderRajuB) [reply tests/test_cms_menus.py]: renamed to navigation_breadcrumb

File: djangocms_navigation/templatetags/navigation_menu_tags.py [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+from classytags.arguments import Argument
+from classytags.core import Options
+from classytags.helpers import InclusionTag
+
+from cms.models import Page
+
+register = template.Library()
+
+
+class ShowBreadcrumb(InclusionTag):
```
[the remark this answers is not in the document]
Comment (NarenderRajuB) [reply djangocms_navigation/templatetags/navigation_menu_tags.py]: Updated please review

File: djangocms_navigation/templatetags/navigation_menu_tags.py [hunk 4]
```diff
@@ -0,0 +1,83 @@
...
+
+from classytags.arguments import Argument
+from classytags.core import Options
+from classytags.helpers import InclusionTag
+
+
+register = template.Library()
+
+
+class Navigation_ShowBreadcrumb(InclusionTag):
```
Comment (Aiky30) [inline djangocms_navigation/templatetags/navigation_menu_tags.py:15]: Violates PEP 008 naming where a class name should be upper cased words. Needs to be changed to: NavigationBreadcrumb

File: tests/test_cms_menus.py [hunk 5]
```diff
@@ -810,6 +810,106 @@ def test_basic_projects_softroot_rendering_nodes(self):
...
+        Tree in fixture :
+            menuroot
+               aaa ( Home page Node)
+                   aaa1
+                       ccc
+                           ddd
+                   aaa2
+               bbb
+        Expected result:
+            show_navigation_breadcrumb will return all the parents of the selected navigation node and home page
```
Comment (Aiky30) [inline tests/test_cms_menus.py:824]: The tag name is: navigation_breadcrumb

**Review by Aiky30 — CHANGES_REQUESTED** [review]

**Review by Aiky30 — APPROVED** [review]

**Pull request merged**